c++/examples/h5tutr_extend.cpp

```diff
@@ -113,6 +113,7 @@ main(void)
         // Get information to obtain memory dataspace.
         rank            = filespace->getSimpleExtentNdims();
         herr_t status_n = filespace->getSimpleExtentDims(dimsr);
```

Comment:
This seems awkward.  How about eliminating the variable entirely and just casting the return value from the call to void?

Reply:
Could do. As I said in the commit message though, it's sometimes nice in the debugger to be able to inspect the variable, which I assume is why the variable is there.  But I'm happy to remove it if you prefer.

Reply:
I would also prefer casting to void as a library pattern. These are also examples, so debugging is less of an issue here.

Reply:
Done.